package.json

```diff
@@ -29,15 +29,15 @@
   "main": "lib/index",
   "readmeFilename": "README.md",
   "dependencies": {
-    "lodash": "3.10.x",
+    "lodash": "^4.17.10",
```

Comment:
Updating lodash seems to break 33 out of 145 tests

Reply:
See [here](https://github.com/z0mt3c/node-restify-validation/pull/87)